### PR TITLE
Add a newsletter text to thankyou page

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
As a part of #868 I need to add a custom text to the thankyou page if a newsletter signup is being made. (Thanks, still need to confirm blabla).

I made this seperate PR to be able to test it!

### CODE REVIEW

**Comment by mildaskl** [comment]
The en one. We just sent you -> we've sent you

**Comment by RonaldDijkstra** [comment]
Text is just a placeholder right now, but thank you.

File: source/localizable/bedankt.html.erb [hunk 1]
```diff
@@ -26,14 +26,22 @@ window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function (m, key, value)
...
   if (urlParams.download) {
     // Replace message with download message if it was a download form
     var downloadMessage = '<%= t("thanks.message_download") %>';
-    var messageEl = document.querySelector('body.bedankt .thanks-message');
 
     messageEl.innerHTML = downloadMessage;
     messageEl.querySelector('a').href = urlParams.download;
   }
+
+  if (urlParams.newsletter) {
```
Comment (sn3p) [inline source/localizable/bedankt.html.erb]: Looks good. Could slightly simplify to:

```js
if (urlParams.download) {
  // Replace message with download message if it was a download form
  messageEl.innerHTML = '<%= t("thanks.message_download") %>';
  messageEl.querySelector('a').href = urlParams.download;
}
else if (urlParams.newsletter) {
  // Replace message with newsletter message if it was a newsletter signup
  messageEl.innerHTML = '<%= t("thanks.message_newsletter") %>';
}
```

File: source/localizable/bedankt.html.erb [hunk 2]
```diff
@@ -29,18 +29,13 @@ document.addEventListener('DOMContentLoaded', function() {
...
 
   if (urlParams.download) {
-    // Replace message with download message if it was a download form
-    var downloadMessage = '<%= t("thanks.message_download") %>';
-
-    messageEl.innerHTML = downloadMessage;
-    messageEl.querySelector('a').href = urlParams.download;
+  // Replace message with download message if it was a download form
+  messageEl.innerHTML = '<%= t("thanks.message_download") %>';
+  messageEl.querySelector('a').href = urlParams.download;
```
Comment (sn3p) [inline source/localizable/bedankt.html.erb]: Needs some indents here :octocat:

**Pull request merged**